### ISSUE TITLE
Error shows up every minute

### ISSUE THREAD

**Issue by manar-mk** [avocode/avocode#2025]
---
* **Version**: 2.26.0
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 330573
---
```
TypeError: Cannot read property 'l' of undefined
    at Object.rectToBounds (C:\Users\mk\AppData\Local\avocode\app-2.26.0\resources\app.asar\node_modules\@avocode\stage\dist\utils\rendering.js:1:3087)
    at t.preview (C:\Users\mk\AppData\Local\avocode\app-2.26.0\resources\app.asar\node_modules\@avocode\stage\dist\artboards\artboard.js:1:12044)
    at e.value (file:///C:/Users/mk/AppData/Local/avocode/app-2.26.0/resources/app.asar/dist/avocode.min.js:1:4030358)
    at t.describeLayers (file:///C:/Users/mk/AppData/Local/avocode/app-2.26.0/resources/app.asar/dist/avocode.min.js:1:4074801)
    at e.value (file:///C:/Users/mk/AppData/Local/avocode/app-2.26.0/resources/app.asar/dist/avocode.min.js:1:4245268)
    at e.value (file:///C:/Users/mk/AppData/Local/avocode/app-2.26.0/resources/app.asar/dist/avocode.min.js:1:4243335)
    at t._getSelectionState (file:///C:/Users/mk/AppData/Local/avocode/app-2.26.0/resources/app.asar/dist/avocode.min.js:1:3122535)
    at t.<anonymous> (file:///C:/Use
```

**Comment by manar-mk** [comment]
http://prntscr.com/ji1ccw

**Comment by jakubzitny** [comment]
Hi @manar-mk, thanks for reporting this issue. This was a bug in 2.26.0 version, as already mentioned in #2017. Yesterday, we've released Avocode 2.26.1 with a fix for this bug. Please update and let us know if the problem persists 😉
